examples/src/main/java/StakingExample.java

```diff
@@ -0,0 +1,117 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import com.hedera.hashgraph.sdk.AccountCreateTransaction;
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.hashgraph.sdk.AccountInfo;
+import com.hedera.hashgraph.sdk.AccountInfoQuery;
+import com.hedera.hashgraph.sdk.AccountUpdateTransaction;
+import com.hedera.hashgraph.sdk.Client;
+import com.hedera.hashgraph.sdk.Hbar;
+import com.hedera.hashgraph.sdk.PrecheckStatusException;
+import com.hedera.hashgraph.sdk.PrivateKey;
+import com.hedera.hashgraph.sdk.ReceiptStatusException;
+import com.hedera.hashgraph.sdk.TransactionReceipt;
+import com.hedera.hashgraph.sdk.TransactionResponse;
+import io.github.cdimascio.dotenv.Dotenv;
+
+import java.util.Objects;
+import java.util.concurrent.TimeoutException;
+
+public class StakingExample {
+
+    // see `.env.sample` in the repository root for how to specify these values
+    // or set environment variables with the same names
+    private static final AccountId OPERATOR_ID = AccountId.fromString(Objects.requireNonNull(Dotenv.load().get("OPERATOR_ID")));
+    private static final PrivateKey OPERATOR_KEY = PrivateKey.fromString(Objects.requireNonNull(Dotenv.load().get("OPERATOR_KEY")));
+    // HEDERA_NETWORK defaults to testnet if not specified in dotenv
+    private static final String HEDERA_NETWORK = Dotenv.load().get("HEDERA_NETWORK", "testnet");
+
+    private StakingExample() {
+    }
+
+    public static void main(String[] args) throws TimeoutException, PrecheckStatusException, ReceiptStatusException {
+        Client client = Client.forName(HEDERA_NETWORK);
+
+        // Defaults the operator account ID and key such that all generated transactions will be paid for
+        // by this account and be signed by this key
+        client.setOperator(OPERATOR_ID, OPERATOR_KEY);
+
+        System.out.println("Generating accounts for Staking example...");
+
+        // Create Alice account
+        PrivateKey aliceKey = PrivateKey.generateED25519();
+        AccountId aliceId = new AccountCreateTransaction()
+            .setKey(aliceKey)
+            .setInitialBalance(Hbar.from(10))
+            .execute(client)
+            .getReceipt(client)
+            .accountId;
+        Objects.requireNonNull(aliceId);
+
+        // Create Bob account
+        PrivateKey bobKey = PrivateKey.generateED25519();
+        AccountId bobId = new AccountCreateTransaction()
+            .setKey(bobKey)
+            .setInitialBalance(Hbar.from(10))
+            .execute(client)
+            .getReceipt(client)
+            .accountId;
+        Objects.requireNonNull(bobId);
+
+        printAccountInfo(client, aliceId, "Alice");
+        printAccountInfo(client, bobId, "Bob");
+
+        // Alice stakes hbar to bob
+        AccountUpdateTransaction aliceTx =  new AccountUpdateTransaction()
+            .setAccountId(aliceId)
+            .setProxyAccountId(bobId);
```

Comment:
Proxy account ID was deprecated https://github.com/hashgraph/hedera-protobufs/pull/195. Please use node ID. This feature is available on previewnet. Please test the example works

Reply:
Reference [HIP-406](https://hips.hedera.com/hip/hip-406)

Reply:
you can use `setStakeAccountId` to stake to another account not `setProxyAccountId`

Reply:
I'm rewriting the example.